doc/specs/Keyboard-Selection.md

```diff
@@ -58,21 +61,48 @@ Every combination of the `Direction` and `SelectionExpansionMode` will map to a
 **NOTE**: If `copyOnSelect` is enabled, we need to make sure we update the clipboard on every change in selection.
 
 ### Mark Mode
-Upon mark mode being enabled by a user keybinding (`ToggleMarkMode`), the current cursor position becomes the selection anchor. The arrow keys move the selection anchor by cell. Using the arrow keys while holding `Shift` moves the second selection anchor by cell. Additionally, using the arrow keys while holding `Ctrl` and `Shift` moves the second selection anchor by word. This allows the user to make a selection of their choice. The user can exit mark mode by using the `ToggleMarkMode` keybinding.
+Mark Mode is a mode where the user can create more complete and complex selections using only the keyboard. It can be cycled int an enabled/disabled state through a user keybinding (`toggleMarkMode`).
+
+Upon being enabled, the current cursor position becomes the selection anchor. The user can move the selection anchor using the same keybindings defined for keyboard selection (`moveSelectionAnchor` action).
+
+#### `toggleMarkMode`
+`toggleMarkMode` has two keybinding arguments:
+| Parameter | Accepted Values | Default Value | Description |
+|--|--|--|--|
+| `anchorKeybinding` | regular keybindings and plain modifier keys | `Shift` | The keybinding used to anchor the selection anchor to it's current text buffer position. While holding down that keybinding, any `moveSelectionAnchor` actions will move the other selection anchor (`endSelectionPosition`). |
+| `anchorMode` | `Hold`, `Toggle` | `Hold` | If `Hold`, the user needs to hold the `anchorKeybinding` keybinding to move the second selection anchor. If `Toggle`, the user switches between selection anchors when they press it. |
+

```

Comment:
Okay lemme make sure I've got this right.
* (by default) When the user presses the `toggleMarkMode` binding, and holds <kbd>Shift</kbd>, they'll enter mark mode, and <kbd>Shift+arrow</kbd> keys will move the `endSelectionAnchor`, until <kbd>Shift</kbd> is released, <kbd>Esc</kbd> is pressed, or the user copies.
* If the user changes the `anchorMode` instead to `toggle`, their binding will enter mark mode. <kbd>Shift+Arrows</kbd> will move the cursor, as the `startPosition`. <kbd>Shift</kbd> will toggle the anchor we're moving (which would certainly act weirdly in combination with <kbd>Shift+Arrows</kbd> for navigation, but that's up to the user)

Reply:
I copied your comment and referenced it in #5804. Resolving this for now.